### ISSUE TITLE
Generate PDB files from Foldseek DB

### ISSUE THREAD

**Issue by PawelSzczerbiak** [steineggerlab/foldseek#68]
Do you know how to (efficiently) create PDB files from Foldseek DB having a (long) list of IDs? As I understand, Foldseek DB stores C-alpha atoms only but this is essentially what I need.

The question is somehow related to #15 but I hope it's not a duplicate :)

**Comment by martin-steinegger** [comment]
We have recently added the `--format-mode 5` option to our software, which generates PDB files with all Calpha atoms superimposed based on the aligned coordinates. In order to use this feature, you will need to update your version of foldseek. Please note that the latest version also includes improvements to the hit ordering, so you may notice a change in the order of your hits. If you prefer the old hit order, you can simply sort the results by e-value

**Comment by PawelSzczerbiak** [comment]
Thanks a lot for a quick reply! This feature can be very helpful but the problem I'm facing with is a bit different: I need to create a single PDB file from a Foldseek DB having an identifier or a list of identifiers (no need to create alignments) - just to see how the structures look like.

**Comment by martin-steinegger** [comment]
Do you just want to extract entries to look at them then maybe https://github.com/steineggerlab/foldcomp is the right tool for you?

**Comment by PawelSzczerbiak** [comment]
Yes, that's what I initially wanted to use and probably is the simplest solution. Thanks!
